README.rst

```diff
@@ -66,4 +66,4 @@ from a glob pattern, are copied into the root of the deployment.
 `JinjaExtras` are processed with the Jinja2 template engine after all extras
 are copied to a temporary directory.
 
-`JinjaExtrasContext` is a dictionary passed as the context when Jinja is run.
\ No newline at end of file
+`JinjaExtrasContext` is a dictionary passed as the context when Jinja is run.
```

Comment:
Not sure what changed on this line?

Reply:
IntelliJ likes to put a newline at the end of the file if there wasn't one (Emacs likes to do this, too).